opm/grid/cpgpreprocess/make_edge_conformal.cpp

```diff
@@ -98,10 +98,10 @@ sorted_outer_boundary(const processed_grid&   grid,
             };
         }
 
-        face_edges.emplace_back() = std::array {
+        face_edges.emplace_back(std::array {
```

Comment:
If you write this, you should use `push_back()` instead.  The only reason to use `emplace_back()` is that you can construct the object in place, but for the very specific case of `std::array` this does not work (in C++17) since there are no constructors.  In C++20 you can just pass the elements to `emplace_back()` however.

The current statement is in preparation of C++20.

Reply:
ic. dropped the commit.